ui/components/app/wallet-overview/token-overview.js

```diff
@@ -136,11 +133,7 @@ const TokenOverview = ({ className, token }) => {
       }
       className={className}
       icon={
-        <Identicon
-          diameter={32}
-          address={token.address}
-          image={assetImages[token.address]}
-        />
+        <Identicon diameter={32} address={token.address} image={token.image} />
```

Comment:
Line 109 has `token?.image`, but not here; should it be the same in both places?

Reply:
good catch. I'll remove the optional chaining, since we infer and rely on the fact that token object is a defined object throughout this component.

Reply:
✅